tool/teleport-update/main.go

```diff
@@ -187,7 +187,8 @@ func Run(args []string) int {
 		Envar(autoupdate.SetupVersionEnvVar).StringVar(&ccfg.ForceVersion)
 	setupCmd.Flag("flag", "Use the provided flags to generate configuration files.").
 		Envar(autoupdate.SetupFlagsEnvVar).StringsVar(&ccfg.ForceFlags)
-	setupCmd.Flag("selinux-ssh", "Install the SELinux module for Teleport SSH.").Hidden().BoolVar(&ccfg.SELinuxSSH)
+	setupCmd.Flag("selinux-ssh", "Install the SELinux module for Teleport SSH.").
+		Hidden().Envar(autoupdate.SetupSELinuxSSHEnvVar).BoolVar(&ccfg.SELinuxSSH)
```

Comment:
There may be a subtle issue here: if the teleport-update process invoking `teleport-update setup` has the env var set, it will be set for the subprocess even if `ReexecSetup` is invoked with `false`. Does this matter?

Reply:
Good catch, always setting the env var when reexec'ing seems to fix it, as the flag should take precedence anyways